src/bin/driver.rs

```diff
@@ -119,6 +119,28 @@ impl Default for Settings {
     }
 }
 
+#[derive(Serialize, Copy, Clone, Default, Debug)]
+pub struct Monitor {
+    output_current: [f32; 2],
```

Comment:
Please check the names. Do you need `output` in all these (and elsewhere)? If there is `output`, why is there no `input` but `feedback`? `feedback` is also probably a bad name as that input it does not necessarily need to be for feedback at all.

Reply:
Ok, I mean yes there ate inputs on stabilizer. Those might not be exposed in telemetry but I thought just `voltage` might be confusing for someone who has not used Drive+Stabilizer before.

Reply:
Then call the struct `DriverMonitor` to make that clear. And use two of them, not one with two arrays.